tests/xtro-sharpie/macOS-WebKit.todo

```diff
@@ -1,21 +1 @@
 !missing-protocol-conformance! WKWebView should conform to NSTextFinderClient (defined in 'WKNSTextFinderClient' category)
```

Comment:
Any pointers on what needs to be done to make `WKWebView` conform? I noted that AppKit has several ignores like `tests/xtro-sharpie/macOS-AppKit.ignore:!incorrect-protocol-member! NSTextFinderClient::scrollRangeToVisible: is OPTIONAL and should NOT be abstract` but this seems to be a different case...

Reply:
`NSTextView` conforms to `NSTextFinderClient` so it should be similar (binding wise).

The main difference will be that this is for macOS only (so `#if MONOMAC` will be needed in order to compile this for XI)

The other warnings are likely old binding mistakes or because Apple changed them from `@required` to `@optional` (both happens).